ereuse_devicehub/resources/metric/views.py

```diff
@@ -0,0 +1,39 @@
+from flask import request, g, jsonify
+from ereuse_devicehub.resources.action import schemas
+from teal.resource import View
+
+from ereuse_devicehub.resources.action.models import Allocate, Live, Action, ToRepair, ToPrepare
+from ereuse_devicehub.resources.device import models as m
+from ereuse_devicehub.resources.metric.schema import Metric
+
+
+class MetricsView(View):
+    def find(self, args: dict):
+
+        metrics = {
+                "allocateds": self.allocated(),
+                "live": self.live(),
+        }
+        return jsonify(metrics)
+
+    def allocated(self):
+        # TODO @cayop we need uncomment when the pr/83 is approved
+        # return m.Device.query.filter(m.Device.allocated==True, owner==g.user).count()
+        return m.Device.query.filter(m.Device.allocated==True).count()
+
+    def live(self):
+        # TODO @cayop we need uncomment when the pr/83 is approved
+        # devices = m.Device.query.filter(m.Device.allocated==True, owner==g.user)
+        devices = m.Device.query.filter(m.Device.allocated==True)
+        count = 0
+        for dev in devices:
+            live = dev.last_action_of(Live)
```

Comment:
If you try to run GET of /metrics/ without any previous Action Live, when executing the last_action_of function it returns an error, you would have to add a suppress to set it to 0 in case you don't get any action live.

_Error Output:_ LookupError: <Desktop 1> does not contain actions of types.

Reply:
I build one test with this case and I fixed then.